secretary.py

```diff
@@ -306,20 +306,30 @@ def _prepare_template_tags(self, xml_document):
             parent.removeChild(discard)
 
 
-    def _unescape_entities(self, xml_text):
-        # unescape XML entities gt and lt
+    @staticmethod
+    def _unescape_entities(xml_text):
+        """
+        Strips tags of the form <text:span ...> from inside Jinja elements
+        and unescapes HTML codes for >, <, & and "
+        """
         unescape_rules = {
+            r'(?is)({[{|%].*)(<.?text:s.*?>)(.*[%|}]})': r'\1\3',
```

Comment:
This is now the first rule, which means that the `<text:s...>` tags will get ripped out before the `< > & "` get replaced. In addition I've changed the matching rule to `<.?text:s.*?>` which will non-greedily go to the first `>` that closes the tag, whereas the rule previously was only doing `<.?text:s.?>` which would only match one character after the s.